agot-bg-website/agotboardgame_main/views.py

```diff
@@ -114,7 +114,7 @@ def index(request):
     ]
 
     # Retrieves some stats to show on the front page
-    active_games_count = Game.objects.filter(updated_at__gt=datetime.now() - timedelta(days=2)).count()
+    active_games_count = Game.objects.filter(last_active_at__gt=datetime.now() - timedelta(days=5)).count()
```

Comment:
A Question: What is this `{any_var}_gt` syntax? The last appendix `_gt` means greater than and works together with any var name?

Reply:
Yes, it works with any variable of the model: https://docs.djangoproject.com/fr/4.0/ref/models/querysets/#gt 🙂